pkg/test/resourcefixture/contexts/sql_context.go

```diff
@@ -27,6 +27,12 @@ func init() {
 		ResourceKind:       "SQLInstance",
 	}
 
+	resourceContextMap["postgresinstance"] = ResourceContext{
+		// SQL instances names are reserved for 1 week after use: https://cloud.google.com/sql/docs/mysql/delete-instance
```

Comment:
Looks like the 1-week restriction is no longer there according to the doc. I'd assume skipDriftDetection is not needed?